model/build/build.go

```diff
@@ -81,7 +81,7 @@ func (b *Build) IsFinished() bool {
 // returns boolean to indicate if tasks are complete, string with either
```

Comment:
we should fix the comment to reflect the compile finished case.

Reply:
Fixed.